docs/content/015-api/061-list-nonNull.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: list / nonNull / nullable
+title: list / nonNull
```

Comment:
Since we default to `nonNullDefault: false`, I wanted to avoid advertising `nullable` in the top-level API. It's a bit of an antipattern.